src/search/containers/search-results/search-results.component.ts

```diff
@@ -63,6 +66,45 @@ export class SearchResultsComponent implements OnInit, OnDestroy {
 
     // Hide spinner
     this.showSpinner = false;
+
+    // Update moreResultsToGo flag
+    this.moreResultsToGo = searchResult.resultInfo.moreResultsToGo;
+  }
+
+  public getPreviousResultsPage(): void {
+    // Decrement the start record
+    this.searchService.decrementStartRecord();
+
+    // Get the new start record
+    const startRecord = this.searchService.retrieveState(SearchStatePersistenceKey.START_RECORD);
+    if (startRecord) {
+      this.caseStartRecord = parseInt(startRecord, 10);
+    }
+
+    // Retrieve the search results
+    this.showSpinner = true;
+    this.searchSubscription$ = combineLatest([
```

Comment:
Hi @Daniel-Lam, I think we can take out the common code from getPreviousResultsPage and getNextResultsPage into a separate function.

Reply:
Hi @johnbenjamin-hmcts - makes sense... I will refactor the common lines into a new function.